### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by yangfl**
## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

File: skimage/feature/_texture.pyx [hunk 1]
```diff
@@ -103,7 +103,7 @@ def _local_binary_pattern(double[:, ::1] image,
     image : (N, M) double array
         Graylevel image.
     P : int
-        Number of circularly symmetric neighbour set points (quantization of
+        Number of circular symmetric neighbour set points (quantization of
```
Comment (jni) [inline skimage/feature/_texture.pyx]: I think this should be reverted. "circularly" may not be in standard dictionaries but a quick Google search suggests plenty of references in the technical literature. Same for other corrections of "circularly symmetric".

File: skimage/feature/texture.py [hunk 2]
```diff
@@ -281,7 +281,7 @@ def local_binary_pattern(image, P, R, method='default'):
     image : (N, M) array
         Graylevel image.
     P : int
-        Number of circularly symmetric neighbour set points (quantization of
+        Number of circular symmetric neighbour set points (quantization of
```
Comment (jni) [inline skimage/feature/texture.py]: As above.

File: skimage/restoration/_cycle_spin.py [hunk 3]
```diff
@@ -76,7 +76,7 @@ def cycle_spin(x, func, max_shifts, shift_steps=1, num_workers=None,
     x : array-like
         Data for input to ``func``.
     func : function
-        A function to apply to circularly shifted versions of ``x``.  Should
+        A function to apply to circular shifted versions of ``x``.  Should
```
Comment (jni) [inline skimage/restoration/_cycle_spin.py]: As above.

Comment (jni) [reply skimage/restoration/_cycle_spin.py]:
```suggestion
        A function to apply to circularly shifted versions of ``x``.  Should
```

File: doc/examples/filters/plot_cycle_spinning.py [hunk 4]
```diff
@@ -12,7 +12,7 @@
 averaging the results of the following 3-step procedure for multiple spatial
 shifts, n:
 
-1.) (circularly) shift the signal by an amount, n
+1.) (circular) shift the signal by an amount, n
```
Comment (jni) [inline doc/examples/filters/plot_cycle_spinning.py]:
```suggestion
1.) (circularly) shift the signal by an amount, n
```

File: skimage/feature/_texture.pyx (hunk 1 above)
Comment (jni) [reply skimage/feature/_texture.pyx]:
```suggestion
        Number of circularly symmetric neighbour set points (quantization of
```

File: skimage/feature/texture.py (hunk 2 above)
Comment (jni) [reply skimage/feature/texture.py]:
```suggestion
        Number of circularly symmetric neighbour set points (quantization of
```

**Comment by stefanv** [comment]
@OrkoHunter Take a look at the useful PEP8 report above ;)  Can we strip out the "there are no PEP8 errors" comments?

**Comment by stefanv** [comment]
@yangfl Did you run this through an automated spell checker of sorts?  If so, it would be helpful to make that part of our CI toolchain.

**Comment by yangfl** [comment]
Yes, I use codespell. But as always, spell checkers make false positive, which may be a little bit annoying if added to CI toolchain.

**Pull request merged**